core/test/integration/CMakeLists.txt

```diff
@@ -212,3 +212,4 @@ add_test (NAME ledger-core-integration-CosmosLikeWalletSynchronization.GetSendWi
 add_test (NAME ledger-core-integration-CosmosLikeWalletSynchronization.GetDelegateWithExplorer COMMAND ledger-core-integration-tests --gtest_filter=CosmosLikeWalletSynchronization.GetDelegateWithExplorer)
 add_test (NAME ledger-core-integration-CosmosLikeWalletSynchronization.GetCurrentBlockWithExplorer COMMAND ledger-core-integration-tests --gtest_filter=CosmosLikeWalletSynchronization.GetCurrentBlockWithExplorer)
 add_test (NAME ledger-core-integration-CosmosLikeWalletSynchronization.AllTransactionsSynchronization COMMAND ledger-core-integration-tests --gtest_filter=CosmosLikeWalletSynchronization.AllTransactionsSynchronization)
+add_test (NAME ledger-core-integration-CosmosLikeWalletSynchronization.GasLimitEstimation COMMAND ledger-core-integration-tests --gtest_filter=CosmosLikeWalletSynchronization.GasLimitEstimation)
```

Comment:
The tests added in cosmos_synchronization.cpp aren't all in this list.

(Last time I checked, we used that instead of automatic discovery because of some esoteric Qt::HttpClient related bug)

Reply:
I also forgot to add them, I will do it right now